govcd/sample_govcd_test_config.yaml

```diff
@@ -77,6 +77,15 @@ vcd:
     # An external Network name
     externalNetwork: myexternalnet
```

Comment:
We need to change the description of this field, saying that it is an already existing external network.

Reply:
Fixed